src/OrchardCore.Modules/OrchardCore.Users/Workflows/ViewModels/SelectUsersInRoleTaskViewModel.cs

```diff
@@ -0,0 +1,13 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace OrchardCore.Users.Workflows.ViewModels
+{
+    public class SelectUsersInRoleTaskViewModel
+    {
+        [Required]
+        public string PropertyName { get; set; }
+
+        [Required]
+        public string RoleName { get; set; }
```

Comment:
To use checkboxes, you can define the roles like this

```
public RoleEntry[] Roles { get; set; } = Array.Empty<RoleEntry>();
```

Reply:
Move the the SelectUsersInRole into OC.Roles module instead of users and that would give you access to RoleEntry.

I'll review the changes tomorrow